tests/flow/test_pause_replication_race.py

```
@@ -0,0 +1,154 @@
+
+def _writer(env, idx, create_counts):
+    conn = env.getConnection()
+    while not stop_event.is_set():
+        try:
+            conn.execute_command("GRAPH.QUERY", GRAPH_ID, "CREATE (n:P) RETURN 1")
+            create_counts[idx] += 1
+        except ResponseError as e:
+            if any(msg in str(e) for msg in _EXPECTED_WRITE_ERRORS):
+                continue
+            return
+        except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)